src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/wiremock/PrisonApiMockServer.kt

```diff
@@ -25,14 +25,14 @@ import java.time.LocalDateTime
 class PrisonApiMockServer : WireMockServer(8999) {
   private val gson = getGson()
 
-  fun stubUpdateAttendance(bookingId: Long = 1L, activityId: Long = 2L) {
-    val updateAttendanceUrl = "/api/bookings/$bookingId/activities/$activityId/attendance"
+  fun stubUpdateAttendance(bookingId: Long = 1L, activityId: Long = 2L, status: Int = 201) {
```

Comment:
I was expecting a 200 for an update tbh - does it return a 201 in real life?

Reply:
It does!